homeworks/roman.buzukov_BuzRom/3-async/css/style.css

```diff
@@ -0,0 +1,194 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap");

```

Comment:
Are you sure you need all font weight? If not, we can optimize it and request only the needed one.